### PR TITLE
vendor: update pd to fix a pd-client race

### PR DESCRIPTION
**Pull request by tiancaiamao**
PTAL @disksing @shenli @zimulala @coocood

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -156,6 +156,23 @@ import:
   version: c06e80d9300e4443158a03817b8a8cb37d230320
   subpackages:
   - rate
+- package: golang.org/x/text
```
Comment (shenli) [inline glide.yaml]: Why update this package?

Comment (tiancaiamao) [inline glide.yaml]: No, not "update" it,  I just specify a version.
If this line is not add to the file, every time you run `make update`, 
this package would update to the latest. @shenli

**Comment by shenli** [comment]
LGTM
@coocood @disksing @zimulala PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**